server/docker/Dockerfile

```diff
@@ -29,7 +29,9 @@ RUN apt-get -y update && apt-get -y upgrade && \
     rsync \
     wait-for-it \
     wget \
-    xz-utils
+    xz-utils \
+    zlib1g-dev \
```

Comment:
Please add a comment, that both packages are needed for the armfh architecture.

Reply:
Ok, after some struggle with git I added the required comment...